synapse/handlers/device.py

```diff
@@ -419,13 +420,153 @@ def notify_device_update(self, user_id, device_ids):
             for host in hosts:
                 self.federation_sender.send_device_messages(host)
 
+    @defer.inlineCallbacks
+    def notify_user_signature_update(self, from_user_id, user_ids):
+        """Notify a user that they have made new signatures of other users.
+        """
+
+        position = yield self.store.add_user_signature_change_to_streams(
+            from_user_id, user_ids
+        )
+
+        self.notifier.on_new_event(
+            "device_list_key", position, users=[from_user_id],
+        )
+
+    @measure_func("device.get_user_ids_changed")
```

Comment:
Ignore this function -- it seems to have been accidentally copied (maybe from a bad merge) from line ~95.  Will remove in a future patch.